docs/4.3/kubernetes_ssh.md

```diff
@@ -1,29 +1,25 @@
-# Kubernetes and SSH Integration Guide
+# Teleport Kubernetes Access Guide
 
-Teleport v.3.0+ has the ability to act as a compliance gateway for managing privileged access to Kubernetes clusters. This enables the following capabilities:
+Teleport has the ability to act as a compliance gateway for managing privileged
+access to Kubernetes clusters. This enables the following capabilities:
 
 * A Teleport Proxy can act as a single authentication endpoint for both SSH and
-  Kubernetes. Users can authenticate against a Teleport proxy using Teleport's `tsh login` command and retrieve credentials for both SSH and Kubernetes API.
+  Kubernetes. Users can authenticate against a Teleport proxy using Teleport's
+  [`tsh login`](cli-docs.md#tsh-login) command
+  and retrieve credentials for both SSH and Kubernetes API.
 * Users RBAC roles are always synchronized between SSH and Kubernetes, making
   it easier to implement policies like _developers must not access production
   data_.
 * Teleport's session recording and audit log extend to Kubernetes, as well.
-  Regular `kubectl exec` commands are logged into the audit log and the
-  interactive commands are recorded as regular sessions that can be stored and replayed in the
+  Regular `kubectl exec` commands are logged into the audit log and the interactive
+  commands are recorded as regular sessions that can be stored and replayed in the
   future.
 
-![ssh-kubernetes-integration](img/teleport-kube.png)
-
-This guide will walk you through the steps required to configure Teleport to
-work as a unified gateway for both SSH and Kubernetes. We will cover both the open
-source and enterprise editions of Teleport.
-
-For this guide, we'll be using an instance of Kubernetes running on [Google's GKE](https://cloud.google.com/kubernetes-engine/)
-but this guide should apply with any upstream Kubernetes instance.
-
 ## Teleport Proxy Service
-
-By default, the Kubernetes integration is turned off in Teleport. The configuration setting to enable the integration is the `proxy_service/kubernetes/enabled` setting which can be found in the proxy service section in the `/etc/teleport.yaml` file, as shown below:
+By default, the Kubernetes integration is turned off in Teleport. The configuration
+setting to enable the integration is the `proxy_service/kubernetes/enabled` setting
```

Comment:
This might be better expressed as `proxy_service.kubernetes.enabled`? I've never really known how to refer to nested YAML elements without writing out the full:

```yaml
proxy_service:
  kubernetes:
    enabled: true
```